apps/legal_archive/commands.py

```diff
@@ -425,6 +425,8 @@ class ImportLegalPublishQueueCommand(superdesk.Command):
     ]
 
     def run(self, page_size=None):
+        if not app.config['LEGAL_ARCHIVE']:
```

Comment:
I think it is not needed here, these commands won't registered with `LEGAL_ARCHIVE=0` option